app/src/main/java/com/duckduckgo/app/systemsearch/SystemSearchViewModel.kt

```diff
@@ -30,14 +30,18 @@ import com.duckduckgo.app.onboarding.store.isNewUser
 import com.duckduckgo.app.statistics.pixels.Pixel
 import com.duckduckgo.app.statistics.pixels.Pixel.PixelName.*
 import com.jakewharton.rxrelay2.PublishRelay
+import io.reactivex.Observable
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.Disposable
+import io.reactivex.functions.BiFunction
 import io.reactivex.schedulers.Schedulers
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.launch
 import timber.log.Timber
 import java.util.concurrent.TimeUnit
 
+typealias SystemSearchResult = Pair<AutoCompleteResult, List<DeviceApp>>
```

Comment:
Why this construct over a data class? I've no objections, just wondering if there's an advantage of doing it this way?

Reply:
No, but now that you mention it _I_ have objections 😆I've switched to a data class, neater than a Pair and typealias.